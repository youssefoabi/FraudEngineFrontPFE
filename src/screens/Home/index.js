import { Footer, Screen } from '@cdiscount/backoffice-ui';
import React from 'react';

import HomeNavigation from '../../components/HomeNavigation';
import TitleBar from '../../components/TitleBar';

function Home() {
  return (
    <Screen titlebar={<TitleBar title="Accueil" />} bottombar={<Footer />}>
      <HomeNavigation />
    </Screen>
  );
}

export default Home;
