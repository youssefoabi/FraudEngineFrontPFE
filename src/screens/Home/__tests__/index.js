import { itRendersCorrectly } from '@cdiscount/jest-utils';
import React from 'react';

import Home from '../';

describe(Home.name, () => {
  itRendersCorrectly(<Home />);
});
