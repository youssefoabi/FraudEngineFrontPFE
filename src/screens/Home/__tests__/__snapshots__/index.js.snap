// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<o
  bottombar={
    <p
      backToTopLabel="Haut de page"
    />
  }
  titlebar={
    <withRouter(TitleBar)
      title="Accueil"
    />
  }
>
  <HomeNavigation />
</o>
`;
